ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activitySection.component.ts

```diff
@@ -140,16 +62,22 @@ export class ActivitySectionComponent implements OnInit, OnDestroy {
                     if (block.shown !== element.shown) {
                         block.shown = element.shown;
                         blockVisibilityChanged = true;
+
+                        if (block.blockType === BlockType.Activity && !block.shown) {
+                            this.updateValidationForHiddenEmbeddedActivity(block as ActivityActivityBlock);
```

Comment:
A workaround as to reset validation statuses for hidden embedded activities.

Reply:
probably makes sence to rename the method from updateVisibility to updateVisibilityAndValidation

Reply:
Fixed